### PR TITLE
utils/coreutils: Update to 8.27

### PR DESCRIPTION
**Pull request by diizzyy**
Maintainer: @jow-
Compile tested: ar71xx, TP-Link TL-WDR3600, LEDE trunk
Run tested: Not yet

Description:

Update coreutils to 8.27
Refresh and split patches

Signed-off-by: Daniel Engberg <daniel.engberg.lists@pyret.net>

### CODE REVIEW

**Comment by diizzyy** [comment]
This needs reviewing, I'm not sure if I managed to do this correctly but it compiles fine at least.

**Comment by diizzyy** [comment]
@jow- ping

File: utils/coreutils/patches/003-no_docs_man_tests-Makefile-am.patch [hunk 1]
```diff
@@ -0,0 +1,28 @@
+diff --git a/Makefile.am b/Makefile.am
```
Comment (hauke) [inline utils/coreutils/patches/003-no_docs_man_tests-Makefile-am.patch]: I think you should merge this into the previous patch

**Comment by hauke** [comment]
These utils where added in the last few years, but are not packaged: b2sum base32 df numfmt please add them

**Comment by champtar** [comment]
@diizzyy you need to fix the conflict(s)
@jow- please ack/nack

**Comment by diizzyy** [comment]
I'll just close it, I have no need for it anymore.

**Pull request closed without merging**